docker-compose.yml

```diff
@@ -18,6 +18,7 @@ services:
         environment:
             CRATESFYI_RUSTWIDE_WORKSPACE: /opt/docsrs/rustwide
             CRATESFYI_DATABASE_URL: postgresql://cratesfyi:password@db
+            DOCSRS_STORAGE_BACKEND: s3
```

Comment:
Is the performance benefit visible when using docs.rs interactively?

Reply:
Well this is confusing:
```
$ DOCSRS_STORAGE_BACKEND=database time cargo test
41.17user 3.39system 1:08.27elapsed 65%CPU (0avgtext+0avgdata 327156maxresident)k
0inputs+1536outputs (0major+122784minor)pagefaults 0swaps
$ DOCSRS_STORAGE_BACKEND=s3 time cargo test
35.57user 2.88system 0:22.83elapsed 168%CPU (0avgtext+0avgdata 335388maxresident)k
0inputs+1536outputs (0major+122761minor)pagefaults 0swaps
```
No need to change it, although I'm now wondering if I did something wrong taking the benchmarks.

Reply:
Wait. The environment variable in `docker-compose.yml` is for using docs.rs with `docker-compose up`.

You are not seeing performance differences when running `cargo test` because now `cargo test` *hardcodes* using the database.

Reply:
Ahhh, that makes sense. Then my point was moot in the first place, builds will take much longer than uploading the files. Thanks!